tests/Spout/Writer/ODS/WriterTest.php

```diff
@@ -4,6 +4,7 @@
 
 use Box\Spout\Common\Exception\SpoutException;
 use Box\Spout\Common\Type;
+use Box\Spout\Reader\ReaderFactory;
```

Comment:
Let's avoid using the Reader in the Writer test. Instead, can you add a test in the ODSTest.php file that tests the escaper? Although it won't test the same thing (the test you wrote being a full integration testing, the escaper test being more like a unit test), I think it will be good enough.